lib/components/navbar/NavLink.tsx

```diff
@@ -1,30 +1,52 @@
 import Link from 'next/link';
 import { useRouter } from 'next/router';
 import { LinkProps } from './links';
-import { chakra, Link as ChakraLink } from '@chakra-ui/react';
+import { Box, chakra, Link as ChakraLink, useDisclosure } from '@chakra-ui/react';
 
-const NavLink: React.FC<LinkProps> = ({ path, label }) => {
+const NavLink: React.FC<LinkProps> = ({ path, label, dropDownOptions }) => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
   const router = useRouter();
   const isActive = router.pathname === path;
 
   return (
-    <Link href={path} passHref>
-      <ChakraLink
-        _hover={{ cursor: 'pointer' }}
-        height="fit-content"
-        data-cy={'nav-bar-' + label}
-        color="#58595B"
-        textDecor={isActive ? 'underline' : 'none'}
-        textUnderlineOffset="3px"
-        textDecorationThickness="3px"
-        textDecorationColor={'var(--magenta)'}
-        tabIndex={0}
-        _focus={{ boxShadow: 'none' }}>
-        <chakra.span fontWeight="bold" fontSize="24px">
-          {label}
-        </chakra.span>
-      </ChakraLink>
-    </Link>
+    <Box position="relative" onMouseEnter={onOpen} onMouseLeave={onClose}>
+      <Link href={path} passHref>
+        <ChakraLink
+          _hover={{ cursor: 'pointer' }}
+          height="fit-content"
+          data-cy={'nav-bar-' + label}
+          color="#58595B"
+          textDecor={isActive ? 'underline' : 'none'}
+          textUnderlineOffset="3px"
+          textDecorationThickness="3px"
+          textDecorationColor={'var(--magenta)'}
+          tabIndex={0}
+          _focus={{ boxShadow: 'none' }}>
+          <chakra.span fontWeight="bold" fontSize="24px">
+            {label}
+          </chakra.span>
+        </ChakraLink>
+      </Link>
+      {dropDownOptions && isOpen && (
+        <Box
+          border="3px solid var(--magenta)"
+          position="absolute"
+          width="100%"
+          background="#FFF"
+          zIndex="100">
+          {dropDownOptions.map(({ label, path }) => (
+            <Box
+              key={label}
+              backgroundColor={router.pathname === path ? '#c2c4c8' : undefined}
```

Comment:
Could we take this color out into a global variable so this part is a bit more readable? Maybe gray?